lemur/plugins/lemur_kubernetes/plugin.py

```diff
@@ -26,21 +28,32 @@ def ensure_resource(k8s_api, k8s_base_uri, namespace, kind, name, data):
 
     # _resolve_uri(k8s_base_uri, namespace, kind, name, api_ver=DEFAULT_API_VERSION)
     url = _resolve_uri(k8s_base_uri, namespace, kind)
+    current_app.logger.debug("K8S POST request URL: %s", url)
 
     create_resp = k8s_api.post(url, json=data)
+    current_app.logger.debug("K8S POST response: %s", create_resp)
 
     if 200 <= create_resp.status_code <= 299:
         return None
 
-    elif create_resp.json()['reason'] != 'AlreadyExists':
-        return create_resp.content
+    else:
```

Comment:
This whole blurb  from L39-L45 is a bit wordy. We can make it safer by just changing the original lines to the following:

`elif create_resp.json().get('reason', '') != 'AlreadyExists':
        return create_resp.content`

Reply:
Excellent, I knew there would be a better way.